src/components/dls/QuranWord/QuranWord.tsx

```diff
@@ -49,6 +49,7 @@ const QuranWord = ({ word, font, highlight, allowWordByWord = true }: QuranWordP
   const shouldBeHighLighted = highlight || isTooltipOpened;
   return (
     <div
+      data-word-position={`${word.verseKey}:${word.position}`}
```

Comment:
Isn't this equivalent to `word.location` or there is a difference?

Reply:
yes, almost the same. There's a tiny difference, with word.location sometime we get `2:2:5-2` instead of `2:2:5`